creator/data_templates/mutations/template_version.py

```diff
@@ -0,0 +1,254 @@
+from pprint import pprint
+import graphene
+from graphql import GraphQLError
+from graphql_relay import from_global_id
+from django.db import transaction
+from django.conf import settings
+
+from creator.studies.models import Study
+from creator.data_templates.models import (
+    DataTemplate,
+    TemplateVersion,
+)
+from creator.data_templates.nodes.template_version import TemplateVersionNode
+
+
+def check_studies(study_node_ids):
+    """
+    Check if studies exist and return the Study objects if all studies exist
+
+    Helper function called in create/update template version mutations
+    """
+    # Convert graphql relay ids to primary keys
+    study_ids = []
+    for sid in study_node_ids:
+        _, study_id = from_global_id(sid)
+        study_ids.append(study_id)
+
+    # Check all studies exist
+    studies = Study.objects.filter(pk__in=study_ids).only("kf_id").all()
+    if len(studies) != len(study_ids):
```

Comment:
You may be able to condense these two lines and do slightly more work in the db with this:
```
if Study.objects.filter(pk__in=study_ids).count() == len(study_ids):
```

Reply:
I like that ^ approach for counting, but I'm doing more than counting with this method. If all the studies exist, I want to return the study objects so that in the mutation they can be added to the template version obj